import_ldraw.py

```diff
@@ -820,7 +820,25 @@ def create_model(self, scale, context):
                     bpy.ops.mesh.select_all(action='SELECT')
 
                     # Add small gaps between each brick
-                    bpy.ops.transform.resize(value=(0.99, 0.99, 0.99))
+                    gap_width = 0.3
+                    obj_scale = cur_obj.scale * scale
```

Comment:
The script (mostly) uses `camelCase` format, but with the shorter names you have here (e.g., `fac_x`), perhaps these should remain using `under_scores`. I'll have to think about that.